### PR TITLE
Fixed faker.random.word() returning multiple words

### LINKED ISSUE #602: faker.random.words() returns phrases, not words
```
> faker.random.words(4)
'Home Loan Account quantifying Auto Loan Account Credit Card Account'
```

Technically it's returning phrases, not words. It would be nice if `.words()` actually did return words, and maybe we had another function `.phrase()` and `.phrases()`?

### LINKED ISSUE #661: Random word (singular) sometimes returns more than one word
Since there are two methods .random.word() and .random.words() I would expect the first one to only include a singular word, but it sometimes generates more than one word.

### PR DESCRIPTION
**Pull request by Acanguven**
I know lots of people waiting for this.

**Problem**
`faker.random.word()` returns multiple words instead of returning a single word.

**Fix**
This PR fixes the problem by splitting the result from faker method and returns one of the array elements.

**Issues**
Fixes #661
Fixes #602

### CODE REVIEW

**Comment by Acanguven** [comment]
@marak Any updates?

**Comment by yenereren** [comment]
any updates ? @Marak

**Comment by erdoganoksuz** [comment]
Last week i spent 2 hour to understand why my tests fail.
Then i discern faker.random.word() sometimes returns multiple word.

@Marak If method name **word** instead of **words**, should not returns multiple words.

**Comment by WoLfulus** [comment]
I'm waiting for this too

**Comment by Acanguven** [comment]
At least add some people as maintainer so we can merge useful pull requests like this when you are away.

**Comment by opravil-jan** [comment]
Will be this issue solved soon? It is every paintfull.

File: lib/random.js [hunk 1]
```diff
@@ -202,7 +202,9 @@ function Random (faker, seed) {
 
     // randomly pick from the many faker methods that can generate words
     var randomWordMethod = faker.random.arrayElement(wordMethods);
-    return faker.fake('{{' + randomWordMethod + '}}');
+    var result = faker.fake('{{' + randomWordMethod + '}}');
+
+    return faker.random.arrayElement(result.split(' '));
```
Comment (Khartir) [inline lib/random.js]: This alone won't be enough. I use this code in a project and it just returned `Islands,` (I think). So at least commas have to be trimmed of at the end, I don't know what else.

**Comment by Marak** [comment]
Thank you for your contributions everyone.

I did a quick look and I don't think this will PR will work as expected.

We've got a few other solutions pending for this, so it should be resolved soon.

Closing and locking.

**Pull request closed without merging**